lighthouse-core/audits/content-width.js

```diff
@@ -55,8 +55,8 @@ class ContentWidth extends Audit {
       return '';
     }
 
-    return 'The content scroll size is ' + artifact.scrollWidth + 'px, ' +
-        'whereas the viewport size is ' + artifact.viewportWidth + 'px.';
+    return 'The content scroll size is ' + artifact.innerWidth + 'px, ' +
+        'whereas the viewport size is ' + artifact.outerWidth + 'px.';
```

Comment:
I'm not after anything other than to make the audit have more accurate names than it had before the rename :) I'm honestly not sure what this audit is getting after since we already have the `viewport` audit checking for the most obvious condition under which this one will fail.

Proposal: rename `scrollWidth` variable to `viewportWidth`, rename `viewportWidth` to `windowWidth` and update the description accordingly. Do not change comparison of innerWidth to outerWidth.

Aside, my previous example of `scrollWidth != clientWidth` is actually non-sensical I mistakenly assumed they meant something different on documentElement than regular elements.